.github/workflows/tests.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
       SYMPY_USE_CACHE: "no"
       OPTIONS: ${{ matrix.options }}
       BATCHED: ${{ matrix.batched }}
+      SKIP_DEPENDENCIES: "tensorflow|tensorboard"
```

Comment:
Perhaps this is better called something like `TF_DEPENDENCIES`, since it only lists the dependencies that should be skipped if running `"not tf"`. I think that might make more sense here not to confuse it with skipping other dependencies.

It could be slightly confusing is someone looks at the tests and sees the following (which are running _with_ TensorFlow installed but still lists the environment variables).

![Screen Shot 2022-03-01 at 5 17 49 PM](https://user-images.githubusercontent.com/6934626/156258338-508340f6-a0bb-48e4-a242-11b5fc25f9da.png)